scripts/deploy-with-olm.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
```

Comment:
isn't scripts/deploy-with-olm.sh and scripts/clean-olm.sh more a osp-director-dev-tools thing?

Reply:
@stuggi perhaps it is. Going to merge this as these are useful today. We can maybe move them later